service/git/syncer.go

```diff
@@ -78,7 +78,7 @@ func (s *Syncer) Sync(ctx context.Context,
 }
 
 func (s *Syncer) fetch(ctx context.Context, repoInfo *lookout.RepositoryInfo, r *git.Repository, refspecs []config.RefSpec) (err error) {
-	ctxlog.Get(ctx).Infof("fetching references for repository %s: %v", repoInfo.CloneURL, refspecs)
+	ctxlog.Get(ctx).Debugf("fetching references for repository %s: %v", repoInfo.CloneURL, refspecs)
```

Comment:
could you please check if we have any `info` level message somewhere in the callers that explains downloading?
I remember we make this one as `info` because fetching is a slow operation and it looks like lookoutd just hanged without it.

Reply:
We still have `level=info msg="creating local repository for: .....git"`. But reading again I'm realizing now that message is only logged on repo initialization, the next time we fetch a new PR branch you will not see anything.

Fixed in 7359f3a.